xtext-website/documentation/340_lsp_support.md

```diff
@@ -0,0 +1,197 @@
+---
+layout: documentation
```

Comment:
am not sure if gradle is the right thing to pick.

Reply:
I have change the build system from `Gradle` to `Maven` on the screenshot.